pandas/plotting/_matplotlib/core.py

```diff
@@ -973,7 +976,7 @@ def _plot_colorbar(self, ax: "Axes", **kwds):
 
         if mpl_ge_3_0_0():
             # The workaround below is no longer necessary.
-            return
```

Comment:
Is the comment in the line above still relevant?

```
# The workaround below is no longer necessary.
```

Reply:
I think so, yes, it's just saying that
```python

        points = ax.get_position().get_points()
        cbar_points = cbar.ax.get_position().get_points()

        cbar.ax.set_position(
            [
                cbar_points[0, 0],
                points[0, 1],
                cbar_points[1, 0] - cbar_points[0, 0],
                points[1, 1] - points[0, 1],
            ]
        )
        # To see the discrepancy in axis heights uncomment
        # the following two lines:
        # print(points[1, 1] - points[0, 1])
        # print(cbar_points[1, 1] - cbar_points[0, 1])

        return cbar
```
is no longer necessary if mpl is of a modern enough version